datafiles/static/browse.js

```diff
@@ -303,7 +303,8 @@ const createPaginator = () => {
     pag.appendChild(createPageLink(maxPage));
   }
   const isNowOnLastPage = state.page === maxPage;
-  pag.appendChild(createPrevNext(state.page + 1, isNowOnLastPage, "Next"));
+  
+  if(!isNowOnLastPage) pag.appendChild(createPrevNext(state.page + 1, "Next"));
```

Comment:
I think there is value in showing a disabled button, since it will result in a more consistent layout when viewing different pages. What is the motivation for changing this?

Reply:
I wanted browse.js to match the backend rendering of the paginator used on the recent page. On the backend it was less cumbersome to remove Next or Prev from the HTML conditionally then it was to toggle a disabled attribute since in the XHTML library there was no function for adding no attributes.

I wanted to do something like this: `a ! [href "URL", if hasNext then NoAttr else emptyAttr "disabled"] << "Next"`

That best way I could do it was to either to add a empty class or do disabled=True/False. Which browser defaults consider any presence of the disabled attribute whether True or False as disabled.

I can easily return the behavior though. :100:

Reply:
I added back in showing the disabled buttons with the most recent commit.